Linear_Search/Linear_Search.cpp

```diff
@@ -18,23 +18,27 @@ int Linear_Search(int array[], int size, int desired)
 // Driver Function
 int main()
 {
-    int array[] = {2, 4, 6, 7, 3, 1, 5};
-
-    // Element 4 to be searched
-    if(Linear_Search(array, 7, 4) != -1)
-        cout << "Found" << endl;
-    else
-        cout << "Not Found" << endl;
-
-    //Element 9 to be searched
-    if(Linear_Search(array, 7, 9) != -1)
+    int num;
+    cin >> num;
+	int arr[num];
+	for (int i = 0; i < num; i++) {
+		cin >> arr[i];
+	}
+	int desired;
+	cin >> desired;
+    if(Linear_Search(array, num, desired) != -1)
```

Comment:
please change the name of the function to linearSearch

Reply:
done @sridharjajoo